src/simulation/messaging/messages.rs

```diff
@@ -172,6 +173,18 @@ impl Agent {
         routing_mode: RoutingMode,
     ) -> Agent {
         let plan = Plan::from_io(io_person.selected_plan(), id_mappings, routing_mode);
+
+        if plan.acts.is_empty() {
```

Comment:
does this happen? This should not happen. Agents must have at least one activity.